ReadME.md

```diff
@@ -1,205 +1,54 @@
-The following command in entrypoint.sh brings up a virtual ptyp1 device to respond to relays
-```
-socat -d -d PTY,link=/tmp/ptyp1,raw,echo=0 PTY,link=/tmp/ttyp1,raw,echo=0 &
+```bash
+sudo docker-compose down
 ```
 
-#### Test Boiler  
-The following commands in entrypoint.sh bring up a virtual ptyp0 device and then runs working-sync-server to emulate the boiler.
-```
-socat -d -d PTY,link=/tmp/ptyp0,raw,echo=0 PTY,link=/tmp/ttyp0,raw,echo=0 &
-python2 working-sync-server.py /tmp/ttyp0 &
-```
-Actual device to simulator mappings are as follows, the chronos_config.json needs to be changed on the automation QA server to run as follows:
-Boiler --> /dev/ttyUSB0 --> /tmp/ptyp0
-Chillers --> /dev/ttyACM0 --> /tmp/ptyp1
-
-#### Test water temperature
-The water temperature is provided in Centigrade. In order to provide a test incoming water temperature of 100C, use:
-```
-echo -e "YES\nt=100" > /tmp/water_in
-```
-In order to provide a test out water temperature of 140C, use:
-```
-echo -e "YES\nt=140" > /tmp/water_out
-```
-
-#### Python packages dependencies ####
-
-* Flask
-* pyserial
-* apscheduler
-* pymodbus
-* sqlalchemy
-* python-socketio
-* socketIO_client
-* uwsgi
-
-#### System dependencies ####
-
-* nginx
-* uwsgi-plug-python
-
-#### Hardware dependencies ####
-
-* Raspberry Pi
-* USB to RS4485 adapter such as the GearMo Mini USB to RS485 / RS422 Converter FTDI CHIP
-* Relay array such as the Numato 16 Channel USB Relay Module https://numato.com/product/16-channel-usb-relay-module
-* Two DS18B20 temperature sensors connected to GPIO 
-
-#### Database configuration ####
-
-TODO
-#### Deployment instructions ####
-
-To work with shared log and access to the db file www-data and pi users have to be added in one group.
-Installation script does all required actions.
-
-#### Files locations ####
-
-chronos log directory: `/var/log/chronos`
-
-chronos database directory: `/home/pi/chronos_db`
-
-chronos config path: `/etc/chronos_config.json`
-
-#### Managing ####
-Chronos has a daemon which controlled by the following command:
-
-`# service chronos start|stop|restart`
+## Docker Services
 
-Web UI managed by uwsgi app server:
+The Chronos project consists of the following services managed by Docker Compose:
 
-`# service uwsgi start|stop|restart|reload`
+### Frontend
 
-SocketIO server managing:
+- **Service Name:** `dashboard_frontend`
+- **Port:** `5173`
+- **Access URL:** [http://localhost:5173](http://localhost:5173)
 
-`# service uwsgi-socketio start|stop|restart|reload`
+### Backend
 
-## AUTOMATION & TESTING
+- **Service Name:** `dashboard_backend`
+- **Port:** `5172`
+- **Access URL:** [http://localhost:5172/api](http://localhost:5172/api)
+- **API Documentation:** [http://localhost:5172/docs](http://localhost:5172/docs)
 
-This repo uses a self-hosted git-runner on AWS. The .github/workflows/main.yaml file automatically kicks off a new deployment whenever any code changes have been committed to the master branch.
+### Edge Server
```

Comment:
this is a little off. The edge server is likely not run on the same host as the rest of the stack (though it is included in a docker container for convenience during development). Postgres should be included.